src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonInputFormatter.cs

```diff
@@ -177,6 +184,8 @@ void ErrorHandler(object sender, Newtonsoft.Json.Serialization.ErrorEventArgs ev
 
                         _logger.JsonInputException(eventArgs.ErrorContext.Error);
 
+                        exception = eventArgs.ErrorContext.Error;
```

Comment:
Would we be adding the exception message twice now? One from within this method and from the BodyModelBinder?